packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts

```diff
@@ -32,6 +34,13 @@ export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions)
   };
 
   let isPremiumUsername = false;
+
+  const layoutError = validateBookerLayouts(input?.metadata?.defaultBookerLayouts || null);
+  if (layoutError) {
```

Comment:
There was now backend validation yet for the booker layouts in the update profile handler. There is now.